### DISCUSSION TITLE
GitHub Copilot not working with virtual workspace

### DISCUSSION THREAD

**Discussion by garyzhao63** [community/community#64036] [Copilot Conversations]
### Select Topic Area

Bug

### Body

v1.79.10634 was the last working version, however, I cannot use that version anymore since its outdated.

**Comment by Akash1134** [comment]
Hi @garyzhao63 ,

Could you please elaborate what exactly is the virtual environment you're working with? 

Are you getting any error message while attempting to connect via the latest version?

Here's more details about this recent update: #64011
